### ISSUE TITLE
Feature request: support for easily-typed jazz chord symbols

### ISSUE THREAD

**Issue by chrisfargen** [paulrosen/abcjs#433]
For ease of entry, it would be nice to have a few shortcuts for inputting jazz chord symbols.

- Input lowercase letter `o` for diminished, display as `o` in score
- Input number `0` for half-diminished, display as `ø` in score
- Input carrot `^` for major, display as `∆` in score

[ABC Standard](http://abcnotation.com/wiki/abc:standard:v2.2#chord_symbols) specifies only a limited number of chord symbols:

```
m or min        minor
maj             major
dim             diminished
aug or +        augmented
sus             suspended
7, 9 ...        7th, 9th, etc.
```

I see that the standard has already been extended in abcjs to include symbols such as `°`, `ø`, `∆`, and `-`. Adding support for these three additional symbols (and the related `o7`, `07`, and `^7`) would make writing and transcribing jazz much easier. Moreover, it would be consistent with chord symbol entry in [musescore](https://musescore.org/en/handbook/chord-symbols#chord-symbol-syntax).

As always, thanks for your work, Paul!

**Comment by paulrosen** [comment]
That's a good idea. I think I can add that easily. 

I don't think that will break anyone's code because no one would be constructing chords like that. The only danger is if someone is using the chord mechanism for printing arbitrary text. What if they said something like `"Continue"`? I wouldn't want to change the "o" in that. They should have said `"^Continue"` but I think a lot of ABC isn't written that way.

There is a precedent for doing this since the `b` and `#` symbols are already transformed.

In the meantime, "ease of entry" is in the eye of the beholder. On a mac, those symbols are:
```
ø option-O
° shift-option-8
∆ option-J
```
I have to refigure that out every time I get in music-writing mode, but then I remember for a week.

**Comment by chrisfargen** [comment]
> What if they said something like `"Continue"`? I wouldn't want to change the "o" in that. They should have said `"^Continue"` but I think a lot of ABC isn't written that way.

Yes, I can see how this might lead to unexpected results for abc users not familiar with notating annotations vs gchords. Maybe this could be an option similar to `%%freegchord`? Something like `%%jazzgchord`.

> I have to refigure that out every time I get in music-writing mode, but then I remember for a week.

I have to copy-paste them from a .txt file. Just like Spanish class!

**Comment by paulrosen** [comment]
Fixed in 6.0.0-beta.11.

I changed it for everyone, but I require that character as coming right after the chord name and it can't have a letter following. I think that will keep this from happening to someone who wasn't expecting it.

**Comment by chrisfargen** [comment]
Good solution!
